src/modules/iasql_functions/sql/after_install.sql

```diff
@@ -38,6 +38,9 @@ begin
 end;
 $$;
 
+COMMENT
+  ON FUNCTION iasql_modules_installed () IS '{"description": "Returns list of currently installed modules", "sample_usage": "SELECT * FROM iasql_modules_installed()"}';
```

Comment:
Clever!

Reply:
It was propsed by @yrobla . I also think it's a cool idae.